PurchasesCoreSwift/Logging/LoggerWrapper.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  LoggerWrapper.swift
+//  PurchasesCoreSwift
+//
+//  Created by Tina Nguyen on 12/08/20.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+
+@objc(RCLogIntent) public enum LogIntent: Int {
+    case appleError
+    case info
+    case purchase
+    case rcError
+    case rcPurchaseSuccess
+    case rcSuccess
+    case user
+    case warning
+    
+    func emojis() -> String {
```

Comment:
I wouldn't name this `emojis`, since the important part of it isn't really the emoji - it's the prefix. 
i.e.: we could functionally replace all `‼️` with `IMPORTANT` tomorrow and the code would still make sense, because the emoji is an implementation detail. 
so I'd name this `suffix` so that it also conveys how it's meant to be used. 

for bonus swiftiness, it could also be 
```swift
var suffix: String { 
```
(the rest of the code would remain the same, just a signature change). That way usage doesn't need `()`